Provider/PhpcrMenuProvider.php

```diff
@@ -188,7 +189,11 @@ public function get($name, array $options = array())
      */
     public function has($name, array $options = array())
     {
-        return $this->find($name, $options, false) instanceof NodeInterface;
+        try {
+            return $this->find($name, $options, false) instanceof NodeInterface;
+        } catch (NamespaceException $e) {
```

Comment:
any name is valid. So the has method is not allowed to throw any exception if it does not have this menu. If the naming does not match what it supports, it must return false to let another provider handle it